post-scan-actions/aws-python-promote-or-quarantine/README.md

```diff
@@ -8,8 +8,8 @@ After a scan occurs, this example Lambda function places clean files in one buck
     - Install the AWS command line interface (CLI). See [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) for details.
     - Install GNU Make if you don't want to use the AWS CLI. See [GNU Make](https://www.gnu.org/software/make/) for download information.
 2. **Create two S3 buckets**
-    - Create a 'Promote bucket' to receive clean files.
-    - Create a 'Quarantine bucket' to receive quarantined files.
+    - Create a 'Promote bucket' to receive clean files. Example: `FSS-Promote`.
+    - Create a 'Quarantine bucket' to receive quarantined files. Example: `FSS_Quarantine`.
```

Comment:
```suggestion
    - Create a 'Quarantine bucket' to receive quarantined files. Example: `fss-quarantine`.
```
I think underscores and capital letters are invalid for bucket names.
![image](https://user-images.githubusercontent.com/67313382/90225507-16ce9080-de44-11ea-89d4-3019149cc6c4.png)
![image](https://user-images.githubusercontent.com/67313382/90225563-2c43ba80-de44-11ea-8c86-2988f99fd4f6.png)